opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1562,3 +1589,105 @@ def test_dropped_attributes(self):
         self.assertEqual(2, span.events[0].attributes.dropped)
         self.assertEqual(2, span.links[0].attributes.dropped)
         self.assertEqual(2, span.resource.attributes.dropped)
+
+    def _test_span_limits(
```

Comment:
Is this because of bad rebase? Why is it showing as new addition if not used anywhere in this diff?

Reply:
Yes. I had moved them to the bottom but looks like rebase added them back to the original place? Fixed now.